### PR TITLE
Postconference Workshops

### PR DESCRIPTION
**Pull request by phette23**
Changes in this PR:

- uncomment the 'full' filter on workshops/index.html
- add stub data files for locations and speakers with matching IDs
- add 10 posts one for each workshop

Testing:

- visit /schedule/ link to /workshops/ should show list of workshops now
- filters on /workshops/ should function (there are AM, PM, and Full Day workshops this year)
- workshop cards on that page should be accurate
- clicking into workshop posts should work
- workshop posts should have accurate information (check against shared sheet)
  - location is TBD for all workshops and speaker info is just names for now

### CODE REVIEW

**Comment by phette23** [comment]
I shared the postconference spreadsheet that the posts are derived from with Esme's personal email. If someone else ends up reviewing this PR, let me know and I'll share the postcon sheet with you, too.

**Comment by escowles** [comment]
@phette23 I only see one commit, uncommenting a navigation option in the workshop list. Are there other commits with the other changes you need to add to this PR?

**Comment by phette23** [comment]
@escowles I missed adding the untracked files, I just revised the PR take a look again and it should all be there. Sorry!

File: _data/speakers.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# this lacks most of the fields we use, see _data/examples/speakers.yml
+- id: "andreas-orphanides"
+  name: Andreas Orphanides
+  last: Orphanides
+  img_src: /assets/img/nopics/nopic1.jpg
```
Comment (escowles) [inline _data/speakers.yml:5]: You should leave the img_src attribute blank and let the site auto-pick which one to use.

File: _posts/fromSheetsToJekyll.zip
Comment (escowles) [inline _posts/fromSheetsToJekyll.zip:1]: Looks like a stray file got included.

**Comment by escowles** [comment]
@phette23 This looks good to me — just a couple of things to cleanup.

File: _data/speakers.yml (hunk 1 above)
Comment (phette23) [reply _data/speakers.yml]: Nice! I didn't know it worked like that.

**Comment by phette23** [comment]
@escowles OK made those changes, should be good to go now.

**Review by escowles — APPROVED** [review]

**Pull request merged**